flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1257,6 +1257,9 @@
     "org.texstudio.TeXstudio": {
         "finish-args-flatpak-spawn-access": "the app predates this linter rule"
     },
+    "org.ultimatepp.TheIDE": {
+        "finish-args-flatpak-spawn-access": "the app predates this linter rule"
```

Comment:
what? This is not true. The linter rule exist while the package doesn't yet.

Reply:
@hfiguiere  Can you tell me what should I do?

Reply:
like for any other exception request to put the actual reason, don't you think?

Reply:
Done!